### PR TITLE
chore: readable error for unsupported nodejs version

### PR DESCRIPTION
**Pull request by huochunpeng**

### CODE REVIEW

File: bin/aurelia-cli.js [hunk 1]
```diff
@@ -2,6 +2,15 @@
 'use strict';
 const resolve = require('../lib/resolve');
 
+const nodeVersion = process.versions.node;
+const [major, minor] = nodeVersion.split('.');
+if (major < 8 || (major === '8' & minor < 9)) {
```
Comment (bigopon) [inline bin/aurelia-cli.js]: Doesn't look like major and minor can ever be number, no?

Comment (huochunpeng) [reply bin/aurelia-cli.js]: As long as nodejs follows semver, it is safe.

Comment (huochunpeng) [reply bin/aurelia-cli.js]: sh*t, typo '&'

Comment (huochunpeng) [reply bin/aurelia-cli.js]: surprised the typo didn't fail my tests on various version.

Comment (bigopon) [reply bin/aurelia-cli.js]:
```js
[major, minor] = '8.9'.split('.')
(2) ["8", "9"]
major < 8
false
```

Comment (bigopon) [reply bin/aurelia-cli.js]: Ok, I see. I forgot it's valid JS coercion 👍

Comment (huochunpeng) [reply bin/aurelia-cli.js]: `"8" < 8` is false, what's not right?

File: bin/aurelia-cli.js [hunk 2]
```diff
@@ -2,6 +2,15 @@
 'use strict';
 const resolve = require('../lib/resolve');
 
+const nodeVersion = process.versions.node;
+const [major, minor] = nodeVersion.split('.');
+if (major < 8 || (major === '8' && minor < 9)) {
```
Comment (zewa666) [inline bin/aurelia-cli.js]: var semver = require("semver")

if (semver.lt(process.versions.node, "8.9.0")) {
...

Comment (zewa666) [reply bin/aurelia-cli.js]: Why do by ourselves if semver got us here :)

Comment (huochunpeng) [reply bin/aurelia-cli.js]: Thx!
Question: this is a bin file, can it access aurelia-cli dependencies? I will try it.

Comment (huochunpeng) [reply bin/aurelia-cli.js]: Great! It works.

**Pull request merged**